core/templates/components/common-layout-directives/common-elements/sharing-links.component.html

```diff
@@ -107,18 +107,18 @@
   i.oppia-share-icons.font-small {
     font-size: 1.125rem;
   }
-  i.material-icons.embed-link.share-icons.embed-icon {
-    font-size: 1.8rem;
-    height: 1.75rem;
+  .oppia-sharing-links .embed-icon {
+    font-size: 1.8rem !important;
```

Comment:
Hm, interesting, does this not work without `!important`? If so, let's change the selector back to the previous one (sorry!).. we try to avoid `!important` rules (see [guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#css)).

Ditto below.